service/src/main/java/com/sedliarov/learningtable/service/TeacherService.java

```diff
@@ -18,6 +18,8 @@ public interface TeacherService {
 
   void deleteTeacher(UUID id);
 
+  void deleteTeachers();
```

Comment:
We don't use this in controller and my question, For what we have this method?

Reply:
Done